views/listcourses.php

```diff
@@ -1,18 +1,19 @@
 <?php
-    include_once 'header.php';
+include_once 'header.php';
 ?>
 
 <?php
-    require_once '../models/User.php';
-    require_once '../controllers/ParkingDB.php';
+require_once 'models/User.php';
+require_once 'models/ParkingDB.php';
+require_once 'models/UserService.php';
 
-    $hasCourses = false;
-    $courses = array();
+$hasCourses = false;
+$courses = array();
 
-    if (is_logged_in()) {
-        $courses = User::getCourses($_SESSION['u_id']);
-        $hasCourses = sizeof($courses) > 0;
-    }
+if (is_logged_in()) {
+    $courses = UserService::getCourses($_SESSION['u_id']);
```

Comment:
Should't we be calling the controller instead?

Reply:
Yes, correct. Will fix that.